lms/urls.py

```diff
@@ -98,7 +98,7 @@
     url(r'', include('openedx.features.philu_courseware.urls')),
     url(r'', include('openedx.features.badging.urls')),
     url(r'', include('openedx.features.specializations.urls')),
-    url(r'^marketplace/', include('openedx.features.marketplace.urls')),
+    url(r'^communityhub/', include('openedx.features.marketplace.urls')),
```

Comment:
the app name and file structure are different. I see its not in the AC but would that be a good idea to keep the file structure as it is while changing the app name?
cc: @ahmed-faraz46 @zeshanarifios @emraan004

Reply:
@monofal now we are just changing the URL name , no change in any other files

Reply:
I think we should take the advantage of url names here to keep things simple. IMO keep the url names same to keep them consistent with django app name, template files name and methods names and just change the endpoint from `r'^marketplace/'` to `r'^communityhub/'`.